### DISCUSSION TITLE
Messaging queue module running on deno deploy?

### DISCUSSION THREAD

**Discussion by reggi** [denoland/deno#15737] [General]
I'd be interested in a messaging queue similar to https://github.com/OptimalBits/bull one where I can have a node running on one deno deploy server, and connect to it from a different deno deploy server. Ideally the messaging queue could run on postgres, so I wouldn't need so run any other third party databases specific to the queue. Does something like this exist? Are others interested in this?

**Comment by kitsonk** [comment]
Questions about Deno Deploy should take place in https://github.com/denoland/deploy_feedback/ or in [Discord](https://discord.gg/deno).
